test/onboarding/onboarding.test.js

```diff
@@ -741,7 +718,7 @@ describe('Onboarding', () => {
             const headers = new Headers(headersMap);
             const body = JSON.stringify({
                 type: 'FSPIOP_CALLBACK_URL_TRANSFER_ERROR',
-                value: 'http://dfsp/transfers/dee18631-6c40-438a-bdd0-67bac5e4e1c2/error',
+                value: 'http://dfsp/transfers/{{transferId}}/error',
```

Comment:
Is this correct? Specifically `http://dfsp`?

Reply:
I guess this question applies to the other places where this is present also.

Reply:
@msk- as represented by the given `const dfspCallbackUrl = 'http://dfsp';`, yes.

Reply:
Agh, man, I didn't realise that was in tests. That's embarrassing.